File: 백준/Bronze/10430. 나머지/나머지.py
inp=list(map(int,input().split()))
a=inp[0]
b=inp[1]
c=inp[2]
print((a+b)%c)
print(((a%c)+(b%c))%c)
print((a*b)%c)
print(((a%c)*(b%c))%c)

File: 백준/Bronze/3052. 나머지/나머지.js
let input = require('fs').readFileSync('dev/stdin').toString().split('\n').map(x=>parseInt(x));
let res = [];
for (let i in input) {
    let remainder = input[i] % 42;
    if (!isNaN(remainder) && res.indexOf(remainder + "") === -1) res.push(remainder + "");
}
console.log(res.length);
